doc/pages/upgrade/000-upgrade_index.md

```diff
@@ -0,0 +1,8 @@
+# Upgrading the Operator
```

Comment:
Again, I wouldn't differentiate from Operator and Astarte. Ideally, most people should run a 1:1 match of Astarte Operator in terms of minor versions, as much as we might want to support multiple Astarte releases within the same Operator for X reasons. Let's just make sure the guide covers the Astarte upgrade process.